### PR TITLE
Improved pop-up UI #3

### LINKED ISSUE #3: Improve UI of the add-on popup
* [ ] Clone and run this add-on
* [ ] Find the `popup` related code in the popup folder
* [ ] Use your skill and judgment to improve the UI of the popup

### PR DESCRIPTION
**Pull request by jcads**
Redesigned the radio buttons and added some hover effects.

Here's a screenshot on the changes I've made. 
Screenshot 2020-10-01 09:58:39

Closes #3

### CODE REVIEW

**Comment by devcer** [comment]
@jcads The UI looks good. Can you update the color codes to the tailwind present from this demo? -> https://theme-ui.com/demo

File: popup/popup.html [hunk 1]
```diff
@@ -9,24 +9,28 @@
 
 <body>
   <p>
-    <center>Remind me to drink water for every</center>
+    <center>Rehydrate every</center>
```
Comment (devcer) [inline popup/popup.html:12]: This can be Hydrate every.

Comment (jcads) [reply popup/popup.html:12]: > tailwind

You mean the CSS?

Comment (jcads) [reply popup/popup.html]: Working on it.

**Comment by jcads** [comment]
> @jcads The UI looks good. Can you update the color codes to the tailwind present from this demo? -> https://theme-ui.com/demo

Do you mean the CSS? If yes, I'm on it.

**Comment by jcads** [comment]
This?
Screenshot 2020-10-01 22:13:24

or this one? The one with black text on hover
Screenshot 2020-10-01 22:27:45

I just followed the color scheme on the link you've given.

**Comment by devcer** [comment]
I mean the text should have the text color from the tailwind scheme(#2d3748), highlight color in `Reminder updated to...` should be #0063e1. selected option color should be filled with #2b6cb0. hover color should be #718096. what do you think?

**Comment by jcads** [comment]
Oh makes sense now. Sorry bout that.

**Comment by jcads** [comment]
Let me try it.

**Comment by jcads** [comment]
Screenshot 2020-10-01 23:12:05

**Comment by jcads** [comment]
### Hover
Screenshot 2020-10-01 23:19:51

File: popup/popup.js [hunk 2]
```diff
@@ -11,14 +11,24 @@ window.onload = () => {
...
   var radios = document.forms['time-form'].elements['time'];
+  // Reference for the timeout so we can clear it midway through
+  let timeoutHandle; 
   for (var i = 0, max = radios.length; i < max; i++) {
     radios[i].onclick = function () {
+      if (timeoutHandle) // reset remaining time 
+          window.clearTimeout(timeoutHandle);
+
       let time = this.value;
+      const statusText = document.getElementById('status');
```
Comment (devcer) [inline popup/popup.js:22]: @jcads I think L#22 could be outside the for a loop as this is independent of the for-loop.

**Comment by devcer** [comment]
@jcads The new design looks clean. what do you think? Push the changes. I will test and merge after reviewing.

File: popup/popup.html [hunk 3]
```diff
@@ -9,24 +9,28 @@
 
 <body>
   <p>
-    <center>Remind me to drink water for every</center>
```
Comment (devcer) [inline popup/popup.html:12]: center tag is not supported in HTML 5. I think you can add a `text-align` style in css.

Comment (devcer) [reply popup/popup.html:12]: nvm! we can add it later.

**Review by devcer — APPROVED** [review]

**Pull request merged**